vpr/src/route/route_timing.cpp

```diff
@@ -2180,27 +2096,27 @@ static void prune_unused_non_configurable_nets(CBRR& connections_inf,
 }
 
 // Initializes net_delay based on best-case delay estimates from the router lookahead
-static void init_net_delay_from_lookahead(const RouterLookahead& router_lookahead,
-                                          const Netlist<>& net_list,
-                                          const vtr::vector<ParentNetId, std::vector<int>>& net_rr_terminals,
-                                          NetPinsMatrix<float>& net_delay,
-                                          const RRGraphView& rr_graph,
-                                          bool is_flat) {
+void init_net_delay_from_lookahead(const RouterLookahead& router_lookahead,
```

Comment:
Copy or move these comments to the .h file now.
If you know what some uncommented ones do, comment them too in the .h file.